application/src/main/java/org/togetherjava/tjbot/commands/moderation/BanCommand.java

```diff
@@ -90,7 +90,7 @@ private static RestAction<Boolean> sendDm(User target,
 
         return target.openPrivateChannel()
             .flatMap(channel -> channel.sendMessageEmbeds(ModerationUtils
-                .getModActionEmbed(guild, ACTION_VERB, description, reason, duration)
+                .getModActionEmbed(guild, "Ban", description, reason, duration)
```

Comment:
better. but please put it into a constant in the class, similar to ACTION_VERB. so maybe ACTION_TITLE. same with the other classes, thanks